src/olympia/urls.py

```diff
@@ -28,6 +28,10 @@
     ),
     # Home.
     re_path(r'^$', frontend_view, name='home'),
+    # Abuse.
+    # FIXME: modify nginx config to route /<locale>/abuse/ to addons-server
+    # This will need to not clash with new frontend reporting pages !
+    re_path(r'abuse/', include('olympia.abuse.urls')),
```

Comment:
Well then it would be more annoying, since we don't want them to clash with frontend. Plus, in the case of appeals specifically, they should only be made against abuse decisions...